packages/crud/src/vaadin-crud-dialog.js

```diff
@@ -109,20 +155,36 @@ class CrudDialog extends DialogBaseMixin(OverlayClassMixin(ThemePropertyMixin(Po
     return html`
       <vaadin-crud-dialog-overlay
         id="overlay"
+        popover="manual"
+        .crudElement="${this.crudElement}"
```

Comment:
Could we use `owner` property here instead? It would be aligned with other overlay based components.

Reply:
It was a bit confusing because the owner is usually the overlay wrapper so I wanted it to be more explicit. But I've changed it to reuse `owner` and updated the comment that it should be the CRUD element.

Reply:
Actually that has the side effect of reflecting overlay properties to `vaadin-crud` (only `with-backdrop` in this case). Maybe a bit odd?

Reply:
Hmm, not sure if we want that. It's probably ok to have it there for consistency.

Reply:
OK, updated the snapshot.